.github/workflows/CI.yml

```diff
@@ -0,0 +1,144 @@
+name: CI
+run-name: Auto Continues Integration for betajs-media-components
+
+on:
+  push:
+    branches: [ master ]
+    paths: [ 'src/**', 'tests/**' ]
+  pull_request:
+    branches: [ '*' ]
+    types: [ opened, edited, synchronize, reopened ]
+    paths-ignore: [ 'demos/**' ]
+  workflow_dispatch:
+    inputs:
+      use-cache:
+        description: 'Use cache for the workflow'
+        type: choice
+        default: true
+        options: [ 'true', 'false' ]
+  workflow_call:
+
+jobs:
+  build:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    outputs:
+      node-version: ${{ matrix.node-version }}
+      cache-hit: ${{ steps.node.outputs.cache-hit }}
+      cache-key: ${{ steps.node.outputs.cache-key }}
+      package-cache-dir: ${{ steps.node.outputs.package-cache-dir }}
+      build-artifact: ${{ steps.build-artifact.outputs.artifact-id }}
+    defaults:
+      run:
+        working-directory: .
+        shell: bash
+    strategy:
+      matrix:
+        node-version: [ '21.x' ]
```

Comment:
question: would 22.x make more sense since that'll be long-term support?

Reply:
Sure, I can try. On my side it was 21.x which is also not a stable version. Will upgrade it.